Firestore/core/src/firebase/firestore/util/async_queue.cc

```diff
@@ -28,6 +28,7 @@ namespace util {
 AsyncQueue::AsyncQueue(std::unique_ptr<Executor> executor)
     : executor_{std::move(executor)} {
   is_operation_in_progress_ = false;
+  is_shutting_down_ = false;
```

Comment:
Note: `is_operation_in_progress_` is initialized this way because it is an `std::atomic` and has certain quirks that don't apply to plain booleans. Please just use an in-class initializer for `is_shutting_down_`.

Reply:
Done.